rust/stackablectl/src/cmds/operator.rs

```diff
@@ -130,6 +130,12 @@ pub enum CmdError {
     #[snafu(display("invalid repository name"))]
     InvalidRepoName { source: InvalidRepoNameError },
 
+    #[snafu(display("invalid helm chart version {version:?} (need to be semver)"))]
```

Comment:
suggestion: Slightly reword error message.

 ```suggestion
    #[snafu(display("invalid semantic helm chart version {version:?}"))]
```